x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/index.tsx

```diff
@@ -0,0 +1,144 @@
+};
+
+export function ServiceTabEmptyState({
+  title,
+  content,
+  imgSrc,
+  dismissable,
+  onDissmiss,
+}: ServiceTabEmptyStateProps) {
+  const { core } = useApmPluginContext();
+  const { basePath } = core.http;
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <>
+      <EuiPanel color="subdued" paddingSize="xl" style={{ position: 'relative' }}>
+        <EuiFlexGroup alignItems="center">
+          <EuiFlexItem>
+            <EuiTitle size="s">
+              <h3>{title}</h3>
+            </EuiTitle>
+            <EuiSpacer size="s" />
+            <EuiText size="s">{content}</EuiText>
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="center" gutterSize="m">
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  data-test-subj="ServiceTabEmptyStateAddApmButton"
+                  href={basePath.prepend(addAPMLinkButton.href)}
+                  fill
+                >
+                  {addAPMLinkButton.label}
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  data-test-subj="ServiceTabEmptyStateTryItNowButton"
+                  iconType="launch"
+                  iconSide="right"
+                  href={tryItNowButton.href}
+                  target="_blank"
+                >
+                  {tryItNowButton.label}
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiLink
+                  href={learnMoreLink.href}
+                  target="_blank"
+                  data-test-subj="ServiceTabEmptyStateLearnMoreButton"
+                  external
+                >
+                  {learnMoreLink.label}
+                </EuiLink>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          {!imgSrc && (
+            <EuiFlexItem
+              style={{
+                maxHeight: `${euiTheme.base * 14}px`,
+                overflow: 'hidden',
+                borderRadius: `${euiTheme.border.radius.medium}`,
+                border: `${euiTheme.border.thin}`,
+              }}
+            >
+              <EuiImage src={defaultImgSrc} alt={content} />
+            </EuiFlexItem>
+          )}
+
+          {dismissable && (
+            <EuiButtonIcon
+              style={{
+                position: 'absolute',
+                top: `${euiTheme.size.s}`,
+                right: `${euiTheme.size.s}`,
+              }}
+              data-test-subj="ServiceTabEmptyStateDismissButton"
+              iconType="cross"
+              onClick={onDissmiss}
+            />
+          )}
+        </EuiFlexGroup>
+      </EuiPanel>
+      {imgSrc && (
+        <>
+          <EuiSpacer size="l" />
+          <EuiImage src={imgSrc} alt={content} size="fullWidth" style={{ opacity: 0.2 }} />
```

Comment:
Maybe it's me, but 0.2 makes the image almost invisible. Perhaps 0.4 or 0.5 will look better?

0.2

<img width="1453" alt="image" src="https://github.com/user-attachments/assets/353916b4-dd0a-4314-aa06-207cb79d02e7">

0.4

<img width="1442" alt="image" src="https://github.com/user-attachments/assets/adcb5c86-aa1e-4ae9-b1a1-bd58fb0af031">